tutorials/packaging/debug-your-snap.md

```diff
@@ -0,0 +1,222 @@
+---
+id: debug-your-snap
+summary: We are going to walk though ways to debug a snap.
+categories: packaging
+status: published
+tags: snapcraft, debug
+difficulty: 2
+published: 2018-01-08
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+author: Daniel Lim Wee Soong <weesoong.lim@gmail.com>
+
+---
+
+# Debug your snap
+
+## Overview
+Duration: 1:00
+
+Snaps are Linux packages you can install on a wide range of Linux distributions. They are installed from a centralized store and run in a confined sandbox. 
+
+The trouble with sandboxed apps is that they can be harder to debug that non-sandboxed (traditionally packaged) apps. This tutorial will take readers through common steps to debug a malfunctioning snap.
+
+### What you’ll learn
+
+  - How to debug a snap without confinement.
```

Comment:
I find that saying "without confinement" can be confusing, since the YAML always specifies a confinement type. I suggest using the terms "devmode" and "strict" confinement here, assuming that's correct.